Disable the Nvidia GPU device plugin as this requires NVIDIA drivers.
diff --git a/devices/gpu/nvidia/nvml/driver_default.go b/devices/gpu/nvidia/nvml/driver_default.go
index e67efa22e..215ccfc53 100644
--- a/devices/gpu/nvidia/nvml/driver_default.go
+++ b/devices/gpu/nvidia/nvml/driver_default.go
@@ -1,5 +1,3 @@
-// +build !linux
-
 package nvml

 // Initialize nvml library by locating nvml shared object file and calling ldopen
diff --git a/devices/gpu/nvidia/nvml/driver_linux.go b/devices/gpu/nvidia/nvml/driver_linux.go
index bdd777561..ad68efa32 100644
--- a/devices/gpu/nvidia/nvml/driver_linux.go
+++ b/devices/gpu/nvidia/nvml/driver_linux.go
@@ -1,3 +1,5 @@
+// +build !linux
+
 package nvml

 import (
